### PR TITLE
Add CI support

### PR DESCRIPTION
**Pull request by farski**
This allows CI to test pull requests (like this one) so that one can be sure the acceptance test suite works prior to changes being pushed to the CD pipeline action.

### CODE REVIEW

**Comment by farski** [comment]
https://github.com/PRX/meta.prx.org/issues/56

File: buildspec.yml [hunk 1]
```diff
@@ -1,11 +1,26 @@
...
+version: 0.2
+env:
+  variables:
+    DOVETAIL_HOST: "dovetail.staging.prxu.org"
+    DOVETAIL_PROD_HOST: "dovetail.prxu.org"
+    FEEDER_HOST: "feeder.staging.prx.tech"
+    METRICS_HOST: "metrics.staging.prx.tech"
+    PUBLISH_HOST: "publish.staging.prx.tech"
+    PUBLISH_USER: "test@test.com"
+    PUBLISH_PASS: "test1234"
```
Comment (cavis) [inline buildspec.yml]: Our precious password!  Checked into source control!

(Maybe should use parameterstore on this one, just for good measure).

File: test/upload/signature_test.rb [hunk 2]
```diff
@@ -2,7 +2,7 @@
 
  describe 'upload-signature' do
    let(:connection) do
-     Excon.new("#{CONFIG.UPLOAD_HOST}/prod/signature")
+     Excon.new("#{CONFIG.UPLOAD_HOST}/signature")
```
Comment (cavis) [inline test/upload/signature_test.rb]: How was CD passing, if this was broken?

Comment (farski) [reply test/upload/signature_test.rb]: Unclear. Probably the change to the new upload host never got deployed

**Comment by farski** [comment]
@cavis Up for rereview

**Comment by cavis** [comment]
Looks like this test fail may just require a retry?

**Comment by farski** [comment]
@cavis It had already passed and the commit that failed didn't make any real changes, so I didn't feel like trying to get it to pass again.

**Comment by cavis** [comment]
Gotcha.  Idea-for-when-we-have-unlimited-time: retry button next to failures in the ops-builds channel.

**Comment by farski** [comment]
There's a bunch of cases where that doesn't work they way you'd hope. Worth looking into at some point though

**Pull request merged**